code/style.css

```diff
@@ -1,7 +1,220 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Outfit', sans-serif;
+}
+
+body, h1, h2 {
+  margin: 0;
+}
+
+header {
+  background-image: url(images/veggies-hero.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 50vh;
+  width: 100%;
+  padding: 1rem 1.3rem;
+}  
+
+h1 {
+  text-transform: uppercase;
+  color: #FFF;
+  font-size: 2.7rem;
+  margin: 0;
```

Comment:
I see that you wrote margin: 0; here as well as further up together with the body. Maybe this one could be deleted so you don't repeat yourself.